XMonad/Actions/GridSelect.hs

```diff
@@ -38,6 +38,7 @@ module XMonad.Actions.GridSelect (
     withSelectedWindow,
     bringSelected,
     goToSelected,
+    goToSelectedFiltered,
```

Comment:
Did you mean to export more than just `goToSelectedFiltered`? The changelog message mentions `gridselectWindowFiltered` but that doesn't seem to be exported.